pkg/kv/kvserver/metrics.go

```diff
@@ -489,6 +489,12 @@ var (
 		Measurement: "Events",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaRdbWriteStallNanos = metric.Metadata{
```

Comment:
Consider adding an alerting rule for this metric. I assume SRE (@joshimhoff) would like to alert on write stalls eventually.

https://github.com/cockroachdb/cockroach/blob/af221d485beed8b731b8ee371e6a19c5ba2201be/pkg/kv/kvserver/metric_rules.go#L57-L79